### PR TITLE
Change active werden page

### PR DESCRIPTION
**Pull request by sarunast**
Page Aktiv Werden page restructuring + adding de/se sections
screen shot 2018-02-16 at 10 52 22

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 6244b5a4e16eb06bf83ee1fd3a7bc828ab1c7858

https://deploy-preview-181--masifunde.netlify.com

**Comment by sarunast** [comment]
screen shot 2018-02-16 at 10 52 22

File: api/howToSupport.js [hunk 1]
```diff
@@ -42,14 +41,17 @@ const unwrapVolunteerOpening = volunteerOpening => ({
...
   return {
     ...content,
-    regionalGroups: unwrapRegionalGroups(content && content.regionalGroups),
     section1TeamMember: unwrapTeamMember(content && content.section1TeamMember),
     volunteerOpenings: content && content.volunteerOpenings.map(unwrapVolunteerOpening),
     bannerButtonUrl: unwrapPageUrl(content.bannerButtonUrl),
     section1ReferenceList: unwrapTestimonials(content.section1ReferenceList),
+    section4VolunteerOpenings:
+      content && content.section4VolunteerOpenings.map(unwrapVolunteerOpening),
+    section5ReferenceList: unwrapTestimonials(content.section5ReferenceList),
```
Comment (torkjels) [inline api/howToSupport.js]: I think we need a `content &&` here or a check inside `unwrapTestimonials()`. Both here and 3 lines up.

Comment (sarunast) [reply api/howToSupport.js]: Ohh yeah you are right. I just forgot to put it to my commit 😊

**Pull request merged**